### PR TITLE
The new wrapper; ignore the old pull request

### PR DESCRIPTION
**Pull request by nithin127**

### CODE REVIEW

**Comment by maximecb** [comment]
You have another one-line change to PyTorchObsWrapper, but on a different line...

File: gym_duckietown/wrappers.py [hunk 1]
```diff
@@ -37,10 +37,36 @@ def __init__(self, env=None):
         obs_shape = self.observation_space.shape
         self.observation_space = spaces.Box(
             self.observation_space.low[0,0,0],
-            self.observation_space.high[0,0,0],
+            self.observation_space.high[1,1,1],
```
Comment (bhairavmehta95) [inline gym_duckietown/wrappers.py]: Is there a reason that we're specifying the observation space like this?
Can we just hard code 0 and 255?

Comment (maximecb) [reply gym_duckietown/wrappers.py]: Trying to be more flexible, in case someone ever changed it to [0,1], but yes, we could.

Comment (bhairavmehta95) [reply gym_duckietown/wrappers.py]: No worries then, just wondering. But I'm assuming this is also the line we'd need to change back @nithin127

**Pull request merged**